CLOUD-EDGE-CLIENT_COORDINATION_CG_Proposal

```diff
@@ -0,0 +1,11 @@
+1、Group name
+CLOUD-EDGE-CLIENT COORDINATION COMMUNITY GROUP
+
+2、Group Description
+The mission of this group is to provide mechanisms and interfaces between Central Cloud, Edge-Cloud, and Client for computing workload offloading and orchestration. 
+The typical use cases include AI acceleration, cloud gaming, streaming acceleration, etc. The goal is to design a set of new APIs and mechanisms 
+that enable computing workload offloading and orchestration between Central Cloud, Edge Cloud, and Client. It should leverage existing mechanisms as much as possible 
+and should coordinate with related W3C working groups and other SDOs and open-source communities if necessary.
+
+3、Short name
+CloudEdge
```

Comment:
I'm ok with this personally, but Intel is currently trying to emphasize the role of client more.   So "CloudEdgeClient" would probably make the client-computing group at Intel happier, although it is longer.   It does seem odd to leave out one of three things we are coordinating.  Or we could just use an acronym of the long name, e.g. CEC or CECC (CECCCG may be a mouthful though, so I'm thinking CECCG.)   On reflection, though, an acronym does not have as much impact as an "Explanatory" name so Cloud-Edge-Client CG would be better.   I would avoid names that are words but have no relation to the topic.